L/libcgal_julia/build_tarballs.jl

```diff
@@ -3,61 +3,78 @@
 using BinaryBuilder
 import Pkg: PackageSpec
 
-const name = "libcgal_julia"
-const version = v"0.14"
+name = "libcgal_julia"
+version = v"0.14"
+
+isyggdrasil = get(ENV, "YGGDRASIL", "") == "true"
+rname = "libcgal-julia"
 
 # Collection of sources required to build CGAL
-const sources = [
-    GitSource("https://github.com/rgcv/libcgal-julia.git",
-              "5a880a6ee33c3e4ecf200bcc4b56ca5d7307db9e"),
+sources = [
+    isyggdrasil ?
+        GitSource("https://github.com/rgcv/$rname.git",
+                  "2e4dbb3fbf94e82960764ce6cbea490fcb46a9cd") :
+        DirectorySource(joinpath(ENV["HOME"], "src/github/rgcv/$rname"))
 ]
 
 # Dependencies that must be installed before this package can be built
-const dependencies = [
+dependencies = [
     BuildDependency(PackageSpec(name="Julia_jll", version="v1.4.1")),
 
     Dependency("CGAL_jll"),
     Dependency("libcxxwrap_julia_jll"),
 ]
 
 # Bash recipe for building across all platforms
-const script = raw"""
+jlcgaldir = ifelse(isyggdrasil, rname, ".")
+script = raw"""
 ## pre-build setup
 # exit on error
 set -eu
 
+macosflags=
+case $target in
+  *apple-darwin*)
+    macosflags="-DCMAKE_CXX_COMPILER_ID=AppleClang"
+    macosflags="$macosflags -DCMAKE_CXX_COMPILER_VERSION=10.0.0"
```

Comment:
Strictly speaking, we use the vanilla Clang, not the Apple's one.  Does it matter?

Reply:
Honestly, this is a straight rip-off of what is being done in jlqml in order to shut cmake up about not finding features for the given compiler. So I'm not entirely sure if this would be the proper way to go about it.